.dockerignore

```
@@ -1,2 +1,6 @@
 node_modules
-packages/**/node_modules
\ No newline at end of file
+packages/**/node_modules
+.idea/
+.vscode/w
```

```suggestion
.vscode/
```
This looked like a typo